components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/ActionExecutorServiceImpl.java

```diff
@@ -127,6 +152,17 @@ public ActionExecutionStatus execute(ActionType actionType, String[] actionIdLis
             return execute(action, eventContext);
         } catch (ActionExecutionRuntimeException e) {
             // todo: add to diagnostics
+            if (LoggerUtils.isDiagnosticLogsEnabled()) {
```

Comment:
A particular action can be initiated to run from this method as well. So the log indicating that an action got triggered should be added here as well

Reply:
Updated in